source/includes/_dataset_security.md

```diff
@@ -0,0 +1,339 @@
+# Dataset security
+
+The dataset security is the set of rules that defines who (which users / groups) can access what (which metadata / records / fields) at what frequency (api calls quota) for a given this dataset.
+
+It is defined through 3 variables:
+
+* a general access policy that sets whether the dataset is accessible to anyone or just a few specific users
+* specific rulesets for users and groups
+* a default ruleset that applies to anybody else
+
+### Restricted datasets
+
+If the dataset is [set as restricted](#set-the-general-access-policy), then the dataset will only appear in the catalog for users who have a ruleset declared for them, either directly or through a group. Other users won't have any access to the dataset.
+
+The default security ruleset has no meaning for restricted datasets.
+
+### Standard access datasets
+
+All people having access to the domain will be able to see the dataset in the catalog.
```

Comment:
"**Anyone** having access to the domain will be able to see the dataset in the catalog."